src/test/java/itest/GrafanaDatasourceIT.java

```diff
@@ -60,7 +59,8 @@ void createRequest() {
         req = webClient.get("/api/v1/grafana_datasource_url");
     }
 
-    @Test
+    // Disabled for now due to conflict with UploadRecordingIT; see #229
```

Comment:
Just for ease of later back-reference, in case we ever move repos or something, let's expand out that issue number to a full link.